src/classes/BDI_DataImportService.cls

```diff
@@ -692,8 +692,12 @@ global with sharing class BDI_DataImportService {
         return String.isBlank(paymentStatus) ||
             (dataImport.Payment_Status__c.equalsIgnoreCase(
                 ElevateTransactionStatus.CAPTURED.name()
-            ) || (dataImport.Payment_Method__c == ACH_PAYMENT_METHOD
-                && dataImport.Payment_Elevate_ID__c != null));
+            ) ||
+            (isDryRun && dataImport.Payment_Status__c.equalsIgnoreCase(
```

Comment:
@ptewson-sfdo wanted to call your attention to this change in particular

Reply:
Thanks @dfuller114 - My work is still pointing to the GE_PaymentServices location, but I can't think of a graceful way to cutover before this PR is merged. If this work merges first I will try to update #6348 to use the new enum location!